pdf/pdfcanvas.cpp

```diff
@@ -449,13 +453,12 @@ QSGNode* PDFCanvas::updatePaintNode(QSGNode* node, QQuickItem::UpdatePaintNodeDa
                 }
             }
         }
-        else
+        else if (showPage)
         {
-            if( t->childCount() > 0 )
-            {
-                delete t->firstChild();
-                t->removeAllChildNodes();
-            }
+            QSGSimpleRectNode *bgNode = new QSGSimpleRectNode();
+            t->appendChildNode( bgNode );
+            bgNode->setRect( QRectF(0., 0., page.rect.width(), page.rect.height()) );
+            bgNode->setColor( d->linkColor );
```

Comment:
I agree, I was a bit lazy last Sunday and quite satisfied with the visual result because the link color property is set to Theme.highlightColor plus an alpha… I'll add one specific property which is cleaner than supposing that the link color will always be highlightColor !